libraries/ms-common/inc/adc.h

```diff
@@ -14,47 +14,31 @@ typedef enum {
 } AdcMode;
 
 typedef enum {
-  ADC_CHANNEL_0 = 0,
-  ADC_CHANNEL_1,
-  ADC_CHANNEL_2,
-  ADC_CHANNEL_3,
-  ADC_CHANNEL_4,
-  ADC_CHANNEL_5,
-  ADC_CHANNEL_6,
-  ADC_CHANNEL_7,
-  ADC_CHANNEL_8,
-  ADC_CHANNEL_9,
-  ADC_CHANNEL_10,
-  ADC_CHANNEL_11,
-  ADC_CHANNEL_12,
-  ADC_CHANNEL_13,
-  ADC_CHANNEL_14,
-  ADC_CHANNEL_15,
-  ADC_CHANNEL_TEMP,
+  ADC_CHANNEL_TEMP = 16,
   ADC_CHANNEL_REF,
   ADC_CHANNEL_BAT,
   NUM_ADC_CHANNELS,
 } AdcChannel;
 
-typedef void (*AdcCallback)(AdcChannel adc_channel, void *context);
+// Allows use of either channel or GpioAddress, if not using channel write null
```

Comment:
Is this NULL GpioAddress of NULL context?

Reply:
Sorry, deprecated comment, will change